.env.docker

```
@@ -48,3 +48,8 @@ DC_PPL_ENVIRONMENT=testing
 
 #Meilisearch container
 DC_MEILI_MASTER_KEY=MEILI_MASTER_KEY_FOR_DEVELOPMENT_ONLY
+
+# Payload CMS Admin Redirect
+DC_PAYLOAD_API_KEY=xxx
+DC_PAYLOAD_BASE_URL=http://localhost:8083
+DC_FEATURE_PAYLOAD_ENABLED=0
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)